common/models/workspace.js

```diff
@@ -36,14 +36,15 @@ module.exports = function(Workspace) {
     var TEMPLATE_DIR = path.join(__dirname, '..', '..', 'templates', 'projects');
     var DEFAULT_TEMPLATE = 'api-server';
     var DEPENDENCIES_3_X = {
-      'loopback': '^3.0.0',
+      'loopback': '^3.22.0',
       'loopback-component-explorer': '^4.0.0',
     };
     var DEPENDENCIES_2_X = {
-      'loopback': '^2.22.0',
-      'loopback-component-explorer': '^2.4.0',
+      'loopback': '^2.40.0',
+      'loopback-component-explorer': '^5.4.0',
```

Comment:
DEPENDENCIES_2_X are using versions in Active LTS mode, explorer version 5.x is the right one to use here 👍